src/Cli.elm

```diff
@@ -70,7 +76,15 @@ program =
   --output-dir                       The directory to output to. Defaults to: generated/
 
   --module-name                      The Elm module name. Default to <OAS info.title>
-    
+
+  --effect-types                     Which kind of APIs to generate, the options are:
+                                      - cmd: input -> Cmd msg
+                                      - riskycmd: as above, but using Http.riskyRequest
+                                      - task: input -> Task Http.Error msg
+                                      - riskytask: as above, but using Http.riskyRequest
+                                      - backendtask: for dillonkearns/elm-pages
+                                      - concurrenttask: for andrewMacmurray/elm-concurrent-task
```

Comment:
Nice! ❤️

Reply:
I actually haven't implemented the concurrent task part yet, I should
probably comment it out of the docs